### PR TITLE
Update FreeBSD/OpenBSD

### PR DESCRIPTION
**Pull request by liulitchi**
Sorry for my poor English. If you have found any grammer mistakes, you can re-edit it.

### CODE REVIEW

**Comment by liulitchi** [comment]
add FreeBSD/OpenBSD install instructions.

File: content/introduction/installation.md [hunk 1]
```diff
@@ -83,4 +83,49 @@ Install the MATE Desktop Environment with
 
 - `sudo pacman -Sy mate-desktop-environment`
 {{< /tab >}}
+{{< tab "FreeBSD" >}}
+Install the MATE Desktop Environment with
+- `pkg install xorg lightdm mate`
+
+then enable lightdm to `/etc/rc.conf`:
```
Comment (mbkma) [inline content/introduction/installation.md]: I would write `add` instead of `enable` in all instances. Otherwise its fine.

Comment (ykla) [reply content/introduction/installation.md]: OpenBSD use pkg_add not pkg.

**Comment by liulitchi** [comment]
OK, I will check it

**Comment by mbkma** [comment]
@liulitchi there is a merge conflict now. Can you fix it please?

**Comment by mbkma** [comment]
no respond from PR author. Closing.

**Pull request closed without merging**